includes/Services/FlowService.php

```diff
@@ -23,9 +70,7 @@ class FlowService {
 	public static function get_flow_data() {
```

Comment:
Should we be renaming this to `get_data` as well to stay consistent with the other ones?